data_viz/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Owner\\envs\\ubcsolar\\Scripts\\python.exe"
```

Comment:
Moving forward, should we leave files like these untracked?

Reply:
We should, good catch.